# message describing the processed img info for path following purposes
int32 x_im_midpt
int32 x_midpt_frm_lines
int32 x_im_intercept_l
int32 x_im_intercept_r